### PR TITLE
Add jobs to CI build

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
PR that adds CI-build jobs to this repo

**How to test: **
- [x] open the travis build checks

**Expected outcome: **
- [x] Several build jobs built 
- [x] Screenshot

- [x] reviewed @ingkebil 
- [x] tested
- [x] approved 

This is a minor bump since we add CI func but we don't add any end user func

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,56 @@
...
-cache: pip
+    - "2.7"
+    - "3.6"
 
 services:
-  - mysql
+    - mysql
 
+# command to install dependencies
+# pip locked to version in P_deliver
```
Comment (ingkebil) [inline .travis.yml]: ... locked to P_main?

**Review by ingkebil — APPROVED** [review]
:v:

File: .travis.yml (hunk 1 above)
Comment (patrikgrenfeldt) [reply .travis.yml]: P_deliver is not right, I'll change that, where does it run on python 2.7?

**Comment by patrikgrenfeldt** [comment]
Test results:
image

image
👍

File: .travis.yml (hunk 1 above)
Comment (ingkebil) [reply .travis.yml]: Looks like nowhere. it's python 3.7 for hasta (in P_demux) and python 3.6 for thalamus (in prod)

Comment (patrikgrenfeldt) [reply .travis.yml]: ok, let's change it to prod then and change 2.7 to 3.7 :)

**Pull request merged**